libgearman-server/log.cc

```diff
@@ -504,7 +504,7 @@ gearmand_error_t gearmand_log_gai_error(const char *position, const char *functi
 {
   if (rc == EAI_SYSTEM)
   {
-    return gearmand_log_perror(position, function, errno, message);
+    return gearmand_log_perror(position, function, errno, "getaddrinfo");
```

Comment:
```suggestion
    return gearmand_log_perror(position, function, errno, "%s", message);
```
How about this instead? That should appease CodeQL and doesn't change the behavior on any of the calls.

Reply:
Yeah, that should be fine.

*mumble* Stupid vargs functions.... */mumble*